service/src/main/java/uk/gov/hmcts/reform/fpl/service/PastHearingDatesValidatorService.java

```diff
@@ -25,6 +25,12 @@
             errors.add("Enter a valid end time");
         }
 
+        if (errors.isEmpty() && hearingStartDate != null && hearingEndDate != null
```

Comment:
is the errors.isEmpty() neccassary?

Reply:
yes. we want to validate only when start and end dates are valid.